File: spring-akka-demo/src/main/java/com/example/springakkademo/DemoApplication.java
package com.example.springakkademo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.servlet.ServletComponentScan;

@SpringBootApplication
@ServletComponentScan(value={"com.example.springakkademo.web"})
public class DemoApplication {

	public static void main(String[] args) {
		SpringApplication.run(DemoApplication.class, args);
	}

}

File: group-scala/my-scala-sb/src/main/scala/com/example/myscala/sb/DemoApplication.scala
package com.example.myscala.sb

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
//@EnableEurekaClient
//@EnableDiscoveryClient
class DemoApplication

object DemoApplication {

  def main(args: Array[String]): Unit = {

    SpringApplication.run(classOf[DemoApplication], args: _*)
//    new Test2Consumer().run()
  }

}
